hasher-matcher-actioner/setup.py

```diff
@@ -15,6 +15,7 @@
 extras_require["test"] = sorted({"pytest==6.2.1"} | all_extras)
 extras_require["package"] = ["wheel"]
 extras_require["lint"] = ["black==21.12b0"]
+extras_require["type"] = ["types-requests==2.27.1"]
```

Comment:
If you are adding requests types, why are we ignoring requests.packages.urllib3 in mypy.ini?

Reply:
because the current stub doesn't seem to cover this usage